src/lib/shutdown_manager.js

```diff
@@ -95,7 +95,7 @@ export default class ShutdownManager extends EventEmitter {
   }
 
   scheduleTerminationPoll() {
-    return async () => {
+    return (async () => {
```

Comment:
I don't think you're missing anythign, but I was attempting to do as little changes to things as possible.  Seemed simpler this time around to just add the parens